src/azure-cli-core/azure/cli/core/_config.py

```diff
@@ -22,7 +22,7 @@ class AzConfig(object):
                        '0': False, 'no': False, 'false': False, 'off': False}
 
     def __init__(self):
-        self.config_parser = configparser.SafeConfigParser()
+        self.config_parser = configparser.ConfigParser()
```

Comment:
Bad merge? When I reviewed the PR yesterday, I saw only the `custom.py` got changed